webpack/assets/javascripts/react_app/components/componentRegistry.js

```diff
@@ -8,6 +8,10 @@ import PowerStatus from './hosts/powerStatus/';
 import NotificationContainer from './notifications/';
 import ToastsList from './toastNotifications/';
 import StorageContainer from './hosts/storage/vmware/';
+import AboutComputeTable from './about/compute';
```

Comment:
TBH: I am not too happy about having so many entry level components, is there any chance we can take over multiple tabs in one hoc ? this will allow to have all of the "logic" in one place vs across multiple components